unity-environment/Assets/ML-Agents/Scripts/CoreBrainInternal.cs

```diff
@@ -212,6 +221,26 @@ public void DecideAction()
             runner.AddInput(graph[graphScope + ObservationPlaceholderName[obs_number]][0], observationMatrixList[obs_number]);
         }
 
+        TFTensor[] runned;
```

Comment:
Instead of `runned` it might be better to use a name like `network_output` which is more clear.

Reply:
Otherwise the PR looks good.